docs/components/modeler/web-modeler/advanced-modeling/publish-public-processes.md

```diff
@@ -0,0 +1,139 @@
+import PreviewStartFormImg from '../img/start-form-configured.png';
+import SelectStartEventImg from '../img/select-start-event.png';
+import ImplementModeImg from '../img/implement-mode-active.png';
+import PublicLinkImg from '../img/public-link.png';
+import PublicFormImg from '../img/public-form.png';
+
+<span class="badge badge--cloud">Camunda Platform 8 SaaS only</span>
+
+Camunda Platform 8 SaaS offers a convenient solution for process developers who want to make their processes accessible to users without requiring them to sign up to the platform or understand how tasks and processes work. This feature is particularly useful when you want to gather data or initiate a process from users who are not part of your organization or do not have direct access to Camunda. It also allows you to rapidly test a process with your peers in a development environment.
+
+<img src={PublicFormImg} alt="A public form" />
+
+With this feature, developers can easily publish a process with a form embedded into the start event and enable a public link that can be shared with anyone.
+Users can simply open the link, fill out the form, and submit it to initiate a process instance.
+
+This documentation provides step-by-step instructions on how to leverage public forms to streamline process initiation and to improve adoption of process solutions in your organization.
+
+## Create a public form
+
+To create a form for public access, follow these steps:
+
+1. Navigate back to your project root or folder.
+2. Click on **New**, and select **Form** in the drop down menu.
+   <img src={CreateFormImg} style={{width: 300}} alt="Creating a new form" />
+
+3. Name your form.
+4. Design your form using the **component palette**.
+
+Once ready, return to your process. You can read more about form creation in the [form guide](/docs/guides/utilizing-forms.md).
+
+## Embed the form in the start event
+
+To embed the form in the start event, follow these steps:
+
+1. Make sure you are in the **Implement** mode.
+
+<img src={ImplementModeImg} style={{width: 250}} alt="Active implement mode tab" />
+
+2. Select the start event.
+
+<img src={SelectStartEventImg} style={{width: 800}} alt="Start event of a human workflow" />
+
+3. Ensure that the start event is a [none start event](../../bpmn/none-events/none-events.md#none-start-events). If it isn't, change the start event type accordingly using the **wrench tool**.
+4. Use the blue **form icon** to open the form browser. If the icon does not appear, select the start event again.
+5. Select the form you have created and click on **Embed** to confirm.
+
+<img src={EmbedStartFormImg} style={{width: 400}} alt="Embedding a start form" />
+
+You can preview the embedded form by clicking the **form icon** again.
+
+## Enable the public access
+
+To enable the public access, follow these steps:
+
+1. Open the **Publication** section in the **properties panel** (not the tab of the same name) and activate the toggle.
```

Comment:
> (not the tab of the same name)

What's going on here? Can we make this clearer? Is this a naming issue?

Reply:
@akeller there's a tab in the same screen with the same name, probably we should have named that better (especially for A11y reasons)! I created a Web Modeler issue for that: https://github.com/camunda/web-modeler/issues/5013